pkg/reconciler/taskrun/taskrun.go

```diff
@@ -111,6 +111,11 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 		tr.Status.StartTime = &tr.CreationTimestamp
 	}
 
+	if tr.IsCleanedUp() {
+		c.Logger.Info("Not reconciling cleaned up TaskRun")
```

Comment:
Suggest making this a tad more explicit wrt the specific condition.  Something like:

```go
c.Logger.Infof("Skipping reconcile of TaskRun %q: condition %s is True", tr.Name, v1beta1.ConditionCleanedUp)
```

Reply:
I think this could move to right after the `DeepCopy` in L103

Reply:
I think this could move to right after the `DeepCopy` in L103